styleguide/derek/examples/imac.jade

```diff
@@ -0,0 +1,202 @@
+      .imacBanner-textRow
+        .col-sm-6.imacBanner-textContainer
+          h2.imacBanner-headline.imacBanner-headlineMedium expertise across the leading clouds. <br/>For when your team needs a team
+          .line.left-top
+          .line.left-bottom
+          .line.right-bottom
+      .imacBanner-buttonRow
+        a.button.imacBanner-button Read Our Case Studies
+          i.fa.fa-angle-right.imacBanner-buttonIcon
+
+.half-padding-full
+  .imacBanner.medium
+    .imacBanner-container
+      .imacBanner-textRow
+        .col-sm-6.imacBanner-textContainer
+          h2.imacBanner-headline.imacBanner-headlineMedium the cloud is a mystery wrapped in a riddle. <br/>We can unwrap it for you
+          .line.left-top
+          .line.left-bottom
+          .line.right-bottom
+      .imacBanner-buttonRow
+        a.button.imacBanner-button Read Our Case Studies
+          i.fa.fa-angle-right.imacBanner-buttonIcon
+
+.half-padding-full
+  .imacBanner.medium
+    .imacBanner-container
+      .imacBanner-textRow
+        .col-sm-6.imacBanner-textContainer
+          h2.imacBanner-headline.imacBanner-headlineMedium no one runs openstack better than the ones who invented it
+          .line.left-top
+          .line.left-bottom
+          .line.right-bottom
+      .imacBanner-buttonRow
+        a.button.imacBanner-button Read Our Case Studies
+          i.fa.fa-angle-right.imacBanner-buttonIcon
+
+.half-padding-full
+  .imacBanner.medium
+    .imacBanner-container
+      .imacBanner-textRow
+        .col-sm-6.imacBanner-textContainer
+          h2.imacBanner-headline.imacBanner-headlineMedium YETI “I can’t imagine not having Rackspace managed cloud”
+          .line.left-top
+          .line.left-bottom
+          .line.right-bottom
+      .imacBanner-buttonRow
+        a.button.imacBanner-button Read Our Case Studies
+          i.fa.fa-angle-right.imacBanner-buttonIcon
+
+.half-padding-full
+  .imacBanner.medium
+    .imacBanner-container
+      .imacBanner-textRow
+        .col-sm-6.imacBanner-textContainer
+          h2.imacBanner-headline.imacBanner-headlineMedium Does it feel like the cloud is managing you? <br/>We can Turn the tables
+          .line.left-top
+          .line.left-bottom
+          .line.right-bottom
+      .imacBanner-buttonRow
+        a.button.imacBanner-button Read Our Case Studies
+          i.fa.fa-angle-right.imacBanner-buttonIcon
+
+.half-padding-full
+  .imacBanner.medium
+    .imacBanner-container
+      .imacBanner-textRow
+        .col-sm-6.imacBanner-textContainer
+          h2.imacBanner-headline.imacBanner-headlineMedium spoiler alert: <br/>The cloud isn’t all sunshine and rainbows
+          .line.left-top
+          .line.left-bottom
+          .line.right-bottom
+      .imacBanner-buttonRow
+        a.button.imacBanner-button Read Our Case Studies
+          i.fa.fa-angle-right.imacBanner-buttonIcon
+
+script.
+  $(".imacBanner-headlineLarge").fitText(1.1, { maxFontSize: '45px' });
```

Comment:
The example banners hadn't yet been created when those are called within their ejs files so yes. And the ejs files are included on other pages so should have those calls within them to be modular. As soon as we're all design final (this afternoon) we will have to discuss how we want them included into the www application of this.